src/main/java/org/example/config/SpringConfig.java

```diff
@@ -0,0 +1,21 @@
+package org.example.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+import java.util.Scanner;
+
+@Configuration
+@ComponentScan("org.example")
+@PropertySource("RealEmailNotificationService.properties")
```

Comment:
name of properties file should be from lower case
Suggest just name it application.properties